certify-service/src/main/java/io/mosip/certify/services/VCIssuanceServiceImpl.java

```diff
@@ -164,14 +177,15 @@ private CredentialResponse<?> getCredentialResponse(String format, VCResult<?> v
 
             case "jwt_vc_json-ld":
             case "jwt_vc_json":
+            case VCFormats.MSO_MDOC :
                 CredentialResponse<String> jsonResponse = new CredentialResponse<>();
                 jsonResponse.setCredential((String)vcResult.getCredential());
                 return jsonResponse;
         }
         throw new CertifyException(ErrorConstants.UNSUPPORTED_VC_FORMAT);
     }
 
-    private Optional<CredentialMetadata>  getScopeCredentialMapping(String scope) {
+    private Optional<CredentialMetadata>  getScopeCredentialMapping(String scope, String format) {
```

Comment:
TODO(vharsh): if the `scope` check is exclusively required for LDP_VCs this should get moved to `LdpVcCredentialRequestValidator`. This makes the code cleaner by placing _like with like things_

wdyt @jainhitesh9998, is it okay to share the `issuerMetadata` with the validator impl assuming similar requirements to prop up on Mdoc credential issuance ?

Reply:
Adding my point of view, I think a design change in inji-certify to handle vc issuance for different VC formats would be able to tackle this issue